tests/python/gaia-ui-tests/gaiatest/tests/functional/email/test_bug_1116566.py

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import SkipTest
+from marionette.by import By
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.email.app import Email
+from gaiatest.apps.system.regions.cards_view import CardsView
+
+
+class TestBug1116566(GaiaTestCase):
+    _menu_button_locator = (By.CSS_SELECTOR, '.msg-folder-list-btn')
```

Comment:
I agree that this test case is unusual. I think we can do better here. 

I guess we should use the header locator instead as the problem concerned the whole header. This made me notice that the [header class](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/email/app.py#L199) has access to the entire page. I think we should restrict its scope like we do [here for instance](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py#L124).

In other words, if we restrict the header to a [PageRegion](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/base.py#L133), we would have to define the locator of the header (`By.CSS_SELECTOR, .msg-list-header`) inside the Email page class. Then we would be able to use it in the test this unsual call `len(self.marionette.find_elements(email.header_locator))`.

What do you think about that?